modules/developer_manual/pages/app/fundamentals/info.adoc

```diff
@@ -170,6 +136,37 @@ value is `MyApp`. If a namespace is not provided, the application tries
 to default to the first letter upper-cased application id, e.g., `myapp`
 would be tried under `Myapp`.
 
+== category
+
+The ownCloud Marketplace category where you want to publish the application. 
+The following categories are available:
+
+[width="52%",cols="50%,50%",options="header",]
+|================
+| Category Name | Value to Use
+| Automation | automation
+| Collaboration | collaboration
+| Customization | customization
+| External plugins | external-plugins
+| Games | games
+| Integration | integration
+| Multimedia | multimedia
+| Productivity | productivity
+| Security | security
+| Storage |  storage
+| Tools | tools
+|================
+
+[IMPORTANT] 
+====
+For publishing themes this tag must be present — but empty, as in the example below.
```

Comment:
`this` tag or `the following` tag ?

Reply:
Thanks for spotting that. It's now been updated.